### DISCUSSION TITLE
Limit specific dependencies schedule

### DISCUSSION THREAD

**Discussion by darthmaim** [renovatebot/renovate#30205] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

I'm trying to limit a specific dependency (`renovate` itself in this case - I'm using it to lint the config) to be only updated once per week, because it gets updated quite frequently.

This is the `packageRules` config I tried:
```json
{
  "matchManagers": ["npm"],
  "matchPackageNames": ["renovate"],
  "extends": ["schedule:weekly"]
}
```

But renovate still updates the dependency as soon as there is an update.

Link to full config: https://github.com/GW2Treasures/gw2treasures.com/blob/main/.github/renovate.json
Link to latest PR which was created not `"before 4am on Monday"` (what `schedule:weekly` maps to): https://github.com/GW2Treasures/gw2treasures.com/pull/1365

**Comment by rarkins** [comment]
Use updateNotScheduled

**Reply by darthmaim** [reply]
I don't mind renovate keeping the open PR up-to-date until I merge it, I just want to limit the PR (=branch) creation. Here is the previous PR, and you can see the branch was auto-deleted after merge: https://github.com/GW2Treasures/gw2treasures.com/pull/1362. I'm not sure `updateNotScheduled` is the right config here?

**Reply by RahulGautamSingh** [reply, marked as answer]
You are matching the wrong manager.

**Reply by darthmaim** [reply]
Ah, you are right, it counts as `regex`, not `npm`. I'll try `matchDatasources` instead of `matchManagers`.

**Reply by RahulGautamSingh** [reply]
>  it counts as `regex`

`custom.regex` ✌️
